src/Fantomas.Core.Tests/CodePrinterHelperFunctionsTests.fs

```diff
@@ -3,7 +3,7 @@ module Fantomas.Core.Tests.CodePrinterHelperFunctionsTests
 open NUnit.Framework
 open FsUnit
 open Fantomas.Core.Context
-open Fantomas.Core.FormatConfig
+open Fantomas.Core
 open Fantomas.Core
```

Comment:
Duplicate import here
```suggestion
open Fantomas.Core
```

Reply:
Good spot, there were a couple of those.